app/views/application/_select_language.html.haml

```diff
@@ -0,0 +1,4 @@
+- if locale == :sv
+  = link_to 'English flag', {locale: :en}, class: 'language.en', id: 'change-lang-to-english'
+- else
+  = link_to 'Swedish flag', {locale: :sv}, class: 'language.sv', id: 'change-lang-to-svenska'
```

Comment:
The id's (`change-lang-to-....`) don't seem to be used(?).  If not, should we drop them until and if needed?